src/main/scala/org/hammerlab/guacamole/DistributedUtil.scala

```diff
@@ -312,19 +321,24 @@ object DistributedUtil extends Logging {
     skipEmpty: Boolean,
     function: (Pileup, Pileup) => Iterator[T],
     reference: ReferenceGenome): RDD[T] = {
+
+    val referenceBC = reads1.context.broadcast(reference)
```

Comment:
I don't like removing the Broadcast-less interface, adding marginal complexity to all callers in service of an imagined future use case.

I can add overloaded versions, but that'll be a lot of DistributedUtil boilerplate. I'll try it and see how it looks though.